_content/gateways/start/list.md

```diff
@@ -11,6 +11,10 @@ This is a list of commercial LoRaWAN gateways currently available for purchase.
 
 + [The Things Gateway](../gateway): Following a [successful Kickstarter](https://www.kickstarter.com/projects/419277966/the-things-network) our own LoRa Gateway will soon be available world-wide. At a price of just €300 and with its easy web-based setup, it is the best choice to help build The Things Network.
 
+## Laird - RG1xx
+
+Laird RG186 & RG191 - ( https://www.lairdtech.com/products/rg1xx-lora-gateway ) A cost effective 868 / 915MHz LoRa Gateway with dual band Wi-Fi, Ethernet and BT capabilities with integrated presets for simple one click configuration for The Thing Network
```

Comment:
+ Could you use [Markdown syntax](https://guides.github.com/features/mastering-markdown/) to create the link?
+ Can you check the grammar of the sentence?